assign_rights/views.py

```diff
@@ -1,42 +1,56 @@
-from django.shortcuts import render
 from django.views.generic import CreateView, DetailView, ListView, UpdateView
+from rest_framework.response import Response
+from rest_framework.views import APIView
 
+from .assemble import RightsAssembler
 
-def RightsShellsListView(ListView):
+
+class RightsShellsListView(ListView):
     '''browse and search rights shells'''
     pass
 
 
-def RightsShellsCreateView(CreateView):
+class RightsShellsCreateView(CreateView):
     '''create rights shells'''
     pass
 
 
-def RightsShellsDetailView(DetailView):
+class RightsShellsDetailView(DetailView):
     '''view a rights shell'''
     pass
 
 
-def RightsShellsUpdateView(UpdateView):
+class RightsShellsUpdateView(UpdateView):
     '''update rights shell'''
     pass
 
 
-def GroupingsListView(ListView):
+class GroupingsListView(ListView):
     '''browse and search groupings'''
     pass
 
 
-def GroupingsCreateView(CreateView):
+class GroupingsCreateView(CreateView):
     '''create groupings'''
     pass
 
 
-def GroupingsDetailView(DetailView):
+class GroupingsDetailView(DetailView):
     '''view a grouping'''
     pass
 
 
-def GroupingsUpdateView(UpdateView):
+class GroupingsUpdateView(UpdateView):
     '''update grouping'''
     pass
+
+
+class RightsAssemblerView(APIView):
+    '''
+    Calls the RightsAssembler class from assemblers.
+    '''
+
+    def post(self, request, format=None):
+        request_list = request.data
```

Comment:
I would suggest parsing the request data here, so 
```suggestion
        rights_ids = request.data.get("identifiers")
        start_date = request.data.get("start_date")
        end_date = request.data.get("end_date")
        assembled = RightsAssembler().run(rights_ids, start_date, end_date)
```